### DISCUSSION TITLE
🛫 Release 1.4.0

### DISCUSSION THREAD

**Discussion by valnoxy** [valnoxy/t480-oc#155] [Announcements]
<div align="center">
  
  <br />
      <img src="https://dl.exploitox.de/t480-oc/Hackintosh-T480-Sequoia.png" alt="macOS Sequoia on T480">
  <br />
</div>

## 🔧 Changes
- Added support for macOS Sequoia
- Added support for Broadcom WiFi on Sonoma and newer

> [!IMPORTANT]
> Intel WiFi with Airportitlwm currently doesn't work under macOS Sequoia! Use the HeliPort version instead.

The following base / kexts was updated:
   - **Airportitlwm**: 2.3.0
   - **AppleALC**: 1.9.2
   - **BlueToolFixup**: 2.6.8
   - **CPUFriend**: 1.2.8
   - **IntelBluetoothFirmware**: 2.3.0
   - **IntelBluetoothInjector**: 2.3.0
   - **IntelBTPatcher**: 2.3.0
   - **Lilu**: 1.6.8
   - **NVMeFix**: 1.1.1
   - **WhateverGreen**: 1.6.8
   - **VirtualSMC**: 1.3.3
   - **OpenCore**: 1.0.0

**Full Changelog**: https://github.com/valnoxy/t480-oc/compare/v1.3.0...v1.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/valnoxy/t480-oc/releases/tag/v1.4.0'>🛫 Release 1.4.0</a>.</em>

**Comment by MultimediaLucario** [comment]
I am currently on macOS 14.5 Sonoma and using Itwlm + Heliport due to iServices (iMessage, FaceTime, Find My, etc.) do not work with AirPortItwlm. Does anyone know of a possible way to fix this?

**Comment by MalikBehloul** [comment]
Please Please EFI Os Tahoe 26.0.1 with AppleHDA
